modelbaker/iliwrapper/ili2dbconfig.py

```diff
@@ -339,6 +340,11 @@ def to_ili2db_args(self, extra_args=[], with_action=True):
         if self.db_ili_version is None or self.db_ili_version > 3:
             self.append_args(args, ["--createNlsTab"])
 
+        if self.name_lang:
+            self.append_args(args, ["--nameLang", self.name_lang])
+        elif self.db_ili_version is None or self.db_ili_version > 3:
+            self.append_args(args, ["--nameLang", "NULL"])
```

Comment:
General question:
Why don't we just skip the parameter if `self.name_lang` is not provided?

Reply:
Good question. I think it's time to write some comments here...

Reply:
Added some description. The thing is, that we can add a metaconfig file. The parameters in the metaconfig file are considered by ili2db except when the parameter is in the command line also. So if we have NULL in the nameLang on purpose, we need to add it as NULL, because otherwise it takes the value from the metaconfiguration, what we don't want. 

I hope this makes it clearer. Maybe we should once have a chat about the whole metaconfig stuff...